app/javascript/shared/preact/preact-root-fragment.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * A Preact 11+ implementation of the `replaceNode` parameter from Preact 10.
+ *
+ * This creates a "Persistent Fragment" (a fake DOM element) containing one or more
+ * DOM nodes, which can then be passed as the `parent` argument to Preact's `render()` method.
+ * Source: https://gist.github.com/developit/f4c67a2ede71dc2fab7f357f39cff28c
+ */
+export function createRootFragment(parent, replaceNode) {
```

Comment:
This file is not covered by tests.  Please add tests for this file so as to pass the code-coverage.

Let us know if you need help with tests.